.travis.yml

```diff
@@ -17,7 +18,7 @@ before_install:
 # Install miniconda to avoid compiling scipy
   - mkdir -p download
   - cd download
-  - wget -c https://repo.continuum.io/miniconda/Miniconda2-4.1.11-Linux-x86_64.sh -O miniconda.sh
+  - "[ -f $HOME/download/miniconda.sh ] || wget -c https://repo.continuum.io/miniconda/Miniconda2-4.1.11-Linux-x86_64.sh -O miniconda.sh"
```

Comment:
You don't need to change this line. The wget -c parameter do that. It only redownload if needed.